ui/app/pages/swaps/awaiting-swap/awaiting-swap.js

```diff
@@ -70,8 +70,17 @@ export default function AwaitingSwap ({
 
   let feeinFiat
   if (usedQuote && tradeTxParams) {
-    const renderableGasFees = getRenderableGasFeesForQuote(usedQuote.gasEstimateWithRefund || usedQuote.averageGas, approveTxParams?.gas || '0x0', tradeTxParams.gasPrice, currentCurrency, conversionRate)
-    feeinFiat = renderableGasFees.feeinFiat?.slice(1)
+    const renderableNetworkFees = getRenderableNetworkFeesForQuote(
+      usedQuote.gasEstimateWithRefund || usedQuote.averageGas,
+      approveTxParams?.gas || '0x0',
+      tradeTxParams.gasPrice,
+      currentCurrency,
+      conversionRate,
+      tradeTxParams.value,
+      sourceTokenInfo?.symbol,
+      usedQuote.sourceAmount,
+    )
+    feeinFiat = renderableNetworkFees.feeinFiat?.slice(1)
```

Comment:
Was this an attempt to remove the dollar sign from the value? This will be broken for any locale that represents numbers in some manner other than `<symbol><number>`.

`formatCurrency` returns a string that is meant to be displayed exactly as-is. You'll need to return the number directly if you want to use it without any symbol.

Reply:
I'm noticing now that this was a pre-existing problem. It's definitely a release blocker though - this would give _very_ wrong answers in some cases (e.g. `1,000 $` would turn into `000 $`).

Reply:
I've discovered that this is done in two other places as well, but in all cases the result is used solely for metrics. So maybe not a release blocker then luckily.

Still something we should fix at some point though, so our metrics aren't skewed for other locales.

Reply:
This has been addressed in a separate PR: https://github.com/MetaMask/metamask-extension/pull/9727

There were more problems than just stated in the above comments. In particular, as all of these are just used for our metrics, they should all be denominated in USD.